decompiler/ObjectFile/ObjectFileDB.cpp

```diff
@@ -66,24 +67,24 @@ ObjectFileData& ObjectFileDB::lookup_record(ObjectFileRecord rec) {
  */
 ObjectFileDB::ObjectFileDB(const std::vector<std::string>& _dgos) {
   Timer timer;
-  printf("- Loading Types...\n");
+
+  spdlog::info("-Loading types...");
   dts.parse_type_defs({"decompiler", "config", "all-types.gc"});
 
-  printf("- Initializing ObjectFileDB...\n");
+  spdlog::info("-Initializing ObjectFileDB...");
   for (auto& dgo : _dgos) {
     get_objs_from_dgo(dgo);
   }
 
-  printf("ObjectFileDB Initialized:\n");
-  printf(" total dgos: %d\n", int(_dgos.size()));
-  printf(" total data: %d bytes\n", stats.total_dgo_bytes);
-  printf(" total objs: %d\n", stats.total_obj_files);
-  printf(" unique objs: %d\n", stats.unique_obj_files);
-  printf(" unique data: %d bytes\n", stats.unique_obj_bytes);
-  printf(" total %.1f ms (%.3f MB/sec, %.3f obj/sec)\n", timer.getMs(),
-         stats.total_dgo_bytes / ((1u << 20u) * timer.getSeconds()),
-         stats.total_obj_files / timer.getSeconds());
-  printf("\n");
+  spdlog::info("ObjectFileDB Initialized:");
+  spdlog::info("Total DGOs: {}"), int(_dgos.size());
```

Comment:
should be `spdlog::info("Total DGOs: {}", int(_dgos.size()));`

Reply:
Bruh moment - fixed.